jupyterhub/notebook-images/base/spark-notebook-imagestream.yaml

```diff
@@ -11,7 +12,7 @@ spec:
       openshift.io/imported-from: quay.io/odh-jupyterhub/s2i-spark-minimal-notebook
     from:
       kind: DockerImage
-      name: quay.io/odh-jupyterhub/s2i-spark-minimal-notebook:3.6
-    name: "3.6"
+      name: quay.io/odh-jupyterhub/s2i-spark-minimal-notebook:py36-spark2.4.5-hadoop2.7.3-1
```

Comment:
You can use s2i-spark-minimal-notebook:py36-spark2.4.5-hadoop2.7.3 for the tag name since it will function as latest build of spark2.4.5 and hadoop2.7.3. I was using the -1 as build version number for the notebook to maintain that image hash.

Reply:
Ok